src/c4/ofpstream.hh

```diff
@@ -0,0 +1,86 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   c4/ofpstream.hh
+ * \author Mike Buksas
+ * \brief  Define class ofpstream
+ * \note   Copyright (C) 2016-2018 Los Alamos National Security, LLC.
+ *         All rights reserved.
+ */
+//---------------------------------------------------------------------------//
+
+//---------------------------------------------------------------------------//
+
+#ifndef c4_ofpstream_hh
+#define c4_ofpstream_hh
+
```

Comment:
Add `#include "c4/config.h"` so that `DLL_PUBLIC_c4` is defined (see comments below).